src/main/java/edu/tamu/app/service/manager/GitHubProjectService.java

```diff
@@ -0,0 +1,93 @@
+package edu.tamu.app.service.manager;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHOrganization;
+import org.kohsuke.github.GHProject;
+import org.kohsuke.github.GHProject.ProjectStateFilter;
+import org.kohsuke.github.GHProjectCard;
+import org.kohsuke.github.GHProjectColumn;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+
+import edu.tamu.app.cache.model.Card;
+import edu.tamu.app.cache.model.Member;
+import edu.tamu.app.cache.model.Sprint;
+import edu.tamu.app.model.ManagementService;
+
+public class GitHubProjectService extends AbstractGitHubService {
+
+    public GitHubProjectService(final ManagementService managementService) throws IOException {
+        super(managementService);
+    }
+
+    @Override
+    public List<Sprint> getActiveSprintsByScopeId(final String scopeId) throws Exception {
+        logger.info("Fetching active sprints for remote project with scope id " + scopeId);
+        List<Sprint> activeSprints = new ArrayList<Sprint>();
+        GHRepository repo = github.getRepositoryById(scopeId);
+        List<GHProject> projects = repo.listProjects(ProjectStateFilter.OPEN).asList();
+        for (GHProject project : projects) {
```

Comment:
This could be more elegant by returning the repo list projects method with a filter, map, and collect.